base/sparse/sparsematrix.jl

```diff
@@ -795,7 +795,24 @@ end
 droptol!(A::SparseMatrixCSC, tol, trim::Bool = true) =
     fkeep!(A, (i, j, x) -> abs(x) > tol, trim)
 
+"""
+    dropzeros!(A::SparseMatrixCSC, trim::Bool = true)
```

Comment:
Sorry looks like I should've been more specific. linalg is probably not the best place for these, there is a
```
.. _stdlib-sparse:

Sparse Vectors and Matrices
---------------------------
```
section in `arrays.rst` where these would be a better fit I think